frontend/src/components/Button/Button.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/button-has-type */
+import React from 'react';

```

Comment:
Suggestion - you may skip this import if you only import 'React' from react - since react v17 it's not mandatory:
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports